### PR TITLE
Add a comment for people to be aware of sq-blocking dashboatd in testgrid

### PR DESCRIPTION
**Pull request by krzyzacy**
ref https://github.com/kubernetes/test-infra/pull/1459

cc @fejta

### CODE REVIEW

File: mungegithub/submit-queue/deployment/kubernetes/configmap.yaml [hunk 1]
```diff
@@ -156,6 +156,9 @@ data:
     ci-kubernetes-e2e-gce-1.4-1.5-upgrade-cluster,\
     ci-kubernetes-e2e-gce-1.4-1.5-upgrade-cluster-new,\
     ci-kubernetes-e2e-gce-1.4-1.5-upgrade-master"
+  # SQ-Blocking Jobs
+  # Please also update sq-blocking dashboard in testgrid config for test-infra presubmit to pass (@krzyzacy)
+  # https://github.com/kubernetes/test-infra/blob/master/testgrid/config/config.yaml
```
Comment (apelisse) [inline mungegithub/submit-queue/deployment/kubernetes/configmap.yaml]: Implicit dependencies between files across repos? Yummy :-)

Also, I don't have much knowledge of test-grid config, and I think I would be confused what to do if I had to change it.

> Please also update sq-blocking dashboard in testgrid config

Searching "sq-blocking" in testgrid config ... no match found, wat?

**Comment by krzyzacy** [comment]
@apelisse PR is at https://github.com/kubernetes/test-infra/pull/1459, not merged yet, still debating :-)

**Comment by lavalamp** [comment]
Unassigning myself because I don't know anything about testgrid.

**Comment by krzyzacy** [comment]
@apelisse the dependent PR is merged, I'd better get this comment in here, until we move munger to test-infra some day. Even if people forget to update testgrid config it's trivial for me to fix it up in test-infra.

**Comment by apelisse** [comment]
Yeah, go for it.

**Comment by krzyzacy** [comment]
da label :-)

**Pull request merged**